contents/blog/explore-lineage-api/index.mdx

```diff
@@ -0,0 +1,460 @@
+---
+title: Exploring Lineage History via the Marquez API
+date: 2021-06-25
+author: Michael Collado
+image: ./image.svg
+banner: ./banner.svg
+description: Explore lineage history via marquez
+---
+
+Managing a data pipeline means tracking changes. Sometimes changes to your code, sometimes changes to
+somebody else’s schema, sometimes to the contents of the data itself. Sometimes you need to
+trace the root cause of a problem- somebody changed an int to a string and all the downstream consumers
+broke. Sometimes you want to make a change and see how your consumers were affected- do all the jobs
+run significantly faster after you filter out “unused” records? Or did somebody [rely on those
+"unused" records](https://www.hyrumslaw.com/) to be present in the data? Do the recommendation models
+perform better after you "improved" the data cleaning job upstream? Can you be certain it was your
+change that improved the performance? Sometimes the data itself just looks wrong and you need a way
+to verify that nothing has broken. Why was there a huge drop in traffic to the food delivery site
+yesterday? Was there an outage you didn't hear about? Competitors outbidding your ads? Or did the
+website developers simply stop logging some critical event corrupting every table in your data
+warehouse?
+
+Typically, we think of data lineage in static terms- Job A produces Dataset X, which is consumed by
+Job B which joins it with Dataset Y and produces Dataset Z, which is consumed by&#8230; It’s a map
+that we use to get our heads around the dependencies that exist between the datasets we use to make
+good decisions (how much inventory should I stock in the warehouse to ensure customers get timely
+deliveries?) or to make technical features our customers will love (how can I compile the perfect
+road trip playlist given this customer's listening history?).
+
+But data lineage is much more than a static map of inputs and outputs. _Real time_ lineage and faceted
+metadata give us visibility into how the map changes over time and even allow us to look back in
+history to see how changes in one part of the map cause ripples in other areas. Taking advantage of some
+recent changes to the Marquez API, we’ll demonstrate how to diagnose job failures and how to explore
+the impact of code changes on downstream dependents.
+
+## Getting Started
+
+To get started, we need a running instance of Marquez with a little bit of seed data. For these
+exercises, we'll assume you have a terminal with the following programs installed
```

Comment:
They will also need some form of Docker, like Docker Desktop. They will find out shortly, but being explicit here and linking to it can't hurt. Heck, there might even be someone who hasn't heard of it!